src/main/java/de/tum/in/www1/artemis/web/rest/QuizSubmissionResource.java

```diff
@@ -87,21 +88,27 @@ public QuizSubmissionResource(QuizExerciseRepository quizExerciseRepository, Qui
     }
 
     /**
+     * TODO: Decide if we want to use this endpoint for both submit and save. If so, we may want to use PUT instead of POST
+     * TODO: Don't trust the user submitted values
```

Comment:
IMO having seperate endpoints for submit and save would make sense. Also not trusting user submitted values should definetly be solved in a follow-up.